src/pages/Cart/Cart.tsx

```diff
@@ -0,0 +1,109 @@
+import './cart.css';
+
+import {
+  useEffect,
+  useState,
+} from 'react';
+
+import { getUserCart } from 'apis';
+import { CartCard } from 'components';
+import { useLogin } from 'hooks';
+import { CartProduct } from 'interfaces';
+import { Link } from 'react-router-dom';
+
+export const Cart = () => {
+  const [userCart, setUserCart] = useState<Array<CartProduct>>([]);
+  const [invalidateToggle, setInvalidateToggle] = useState(true);
+  const [price, setPrice] = useState(0);
+  const [discount, setDiscount] = useState(0);
+  const [total, setTotal] = useState(0);
+  const [deliveryCharges, setDeliveryCharges] = useState(0);
```

Comment:
If possible, you can fit all of these into 1 reducer.

Reply:
Yeah I am also thinking about it but I am not completely sure about it. There is only one action I can take as of now and hence I am not much willing. I need to check the comparison and rethink that. May be I'll take this up in a later sprint.
Once this submission is done let's plan a short call where we can discuss it. I would love to hear your opinion on that.